front/reservationitem.php

```diff
@@ -44,6 +44,8 @@
 $res = new ReservationItem();
 $res->display($_GET);
 
+$_POST = $_SESSION['plugin_formcreator']['redirected']['POST'] ?? [];
```

Comment:
Is it possible to go to this page without being redirected ?
In this case $_POST would be erased.

Reply:
Yes this is possible when the user clicks on the menu entry to view the reservations. In this case there is no search to recover. 

I amended my commit to reset the session in this case, to avoid to recover an obsolete search. This is hacky and in some cases this reset will not work.